src/renderer/fonts/font_loader.rs

```diff
@@ -119,12 +120,17 @@ impl FontLoader {
     }
 
     fn load(&mut self, font_key: FontKey) -> Option<FontPair> {
-        let font_style = match (font_key.bold, font_key.italic) {
-            (true, true) => FontStyle::bold_italic(),
-            (false, true) => FontStyle::italic(),
-            (true, false) => FontStyle::bold(),
-            (false, false) => FontStyle::normal(),
-        };
+        let font_style = FontStyle::new(
+            {
+                if font_key.bold {
+                    Weight::BOLD
+                } else {
+                    Weight::NORMAL
+                }
+            },
+            Width::NORMAL,
+            font_key.slant,
```

Comment:
If we already replaced italic from boolean to Slant type, maybe it would make sense to also replace bold from boolean to Weight?

Reply:
yeah, I think it makes sense, just to avoid inconsistencies